src/liberty/libertyProject.ts

```diff
@@ -338,13 +338,21 @@ export class ProjectProvider implements vscode.TreeDataProvider<LibertyProject>
 				if (project.contextValue !== projectType) {
 					project.setContextValue(projectType);
 				}
+				fse.readFile(buildFilePath, "utf8").then(value => {
+					getLabelFromBuildFile( buildFilePath , value).then(projectnew => {
```

Comment:
The `value` parameter (result from `fse.readFile`) should not be passed for Gradle projects. Details discussed over Slack.